packages/bus-rdf-parse/lib/ActorRdfParse.ts

```diff
@@ -84,6 +71,9 @@ export interface IActorRdfParseOutput extends IActorOutput {
   triples?: boolean;
 }
 
+export type IActorRdfParseOutput = IActorParseOutput<RDF.Stream, IActorRdfParseOutputMetadata>;
+
+// TODO: Update these
```

Comment:
Unneeded TODO?

Reply:
Given [this](https://github.com/comunica/comunica/blob/e37d4b7820154e47c9b992f1b23e98e46f99a773/packages/bus-rdf-parse/lib/ActorRdfParseFixedMediaTypes.ts#L20-L22) I assume the fixed media type mediators will be changed/deleted anyway?

Reply:
Probably not in the near future. I had hoped to do this for this major release, but I won't have the time for it.